docs/background.md

```diff
@@ -101,7 +101,7 @@ NativeAppEventEmitter.addListener('healthKit:HeartRate:new', callback);
 ```
 
 When a new sample appears, in order to get the information you need to call
-the [getSamples]('./getSamples().md') methof from your callback function.
+the [getSamples](./getSamples.md) methof from your callback function.
```

Comment:
Just noticed that we also have a typo on the `methof` word, as it should be `method`. Would you mind adding that change to this PR too?

Aside from that, looking great! Thank you for the catch @NS-BOBBY-C !